v4/source/definitions.yaml

```diff
@@ -2751,6 +2751,15 @@ components:
           type: string
         price_per_seat:
           type: string
+    MessagesUsage:
+      type: object
+      properties:
+        enabled:
+          type: boolean
+          description: true, if the cloud limit affects the instance
+        status:
+          type: number
+          description: Usage percentage(0-100) in multiples of 10
```

Comment:
based on @neallred comment in the other thread, this might be higher that 100

Reply:
Yes, and it won't be a usage _percentage_ either anymore. 
It'll be just a Floored/RoundedOff actual no. of posts.